docs/src/main/sphinx/connector/hive.rst

```diff
@@ -459,6 +459,23 @@ with Parquet files performed by the Hive connector.
         definition. The equivalent catalog session property is
         ``parquet_use_column_names``.
       - ``true``
+    * - ``parquet.writer.page-size``
+      - Maximum page size for the Parquet writer.
+      - 1 MB
+    * - ``parquet.writer.block-size``
+      - Maximum block size for the Parquet writer.
+      - 128 MB
+    * - ``parquet.writer.batch-size``
+      - Maximum number of records passed to the writer in each batch.
+      - 10,000
+    * - ``parquet.experimental-optimized-writer.enabled``
+      - Enable optimized Parquet writer which writes from Trino's in-memory data structure
```

Comment:
I don't think we should document experimental feature this like.

Reply:
well .. if we dont document .. how are users supposed to know about it and experiment with it?

Reply:
Using this feature may mean data that is not easily readable by legacy Spark or Hive versions or anything else out there.

Reply:
Okay.. so how do we avoid users finding and using that property. Would it be better to document and mention the problems explicitly?

Reply:
> Okay.. so how do we avoid users finding and using that property.

that's why it's called "experimental".

Reply:
I guess you are saying users should NOT find the property unless they are developers that actually read the log or look at all properties on boot and then read the code. I think that limits the exposure .. and that might be on purpose and also a missed opportunity. This discussion is however larger than the concern in this PR. 

So for this PR .. @findinpath .. just remove the experimental ones..